samtranslator/model/api/api_generator.py

```diff
@@ -1015,7 +1015,7 @@ def _add_models(self) -> None:
 
         self.definition_body = self._openapi_postprocess(swagger_editor.swagger)
 
-    def _openapi_postprocess(self, definition_body: Dict[str, Any]) -> Dict[str, Any]:
+    def _openapi_postprocess(self, definition_body: Dict[str, Any]) -> Dict[str, Any]:  # noqa: too-many-branches
```

Comment:
Is it possible to put in some config file instead?

Reply:
https://github.com/aws/serverless-application-model/blob/6064249125aa1c13d6c219f315f1b3231ff5b248/ruff.toml#L31

That was the previous approach. But it suppresses other functions in the same files